tests/ui/borrowck/argument_number_mismatch_ice.stderr

```diff
@@ -12,6 +12,11 @@ error[E0594]: cannot assign to `*input`, which is behind a `&` reference
    |
 LL |         *input = self.0;
    |         ^^^^^^^^^^^^^^^ `input` is a `&` reference, so the data it refers to cannot be written
+   |
+help: consider changing this to be a mutable referencein the `impl` and the `trait`
```

Comment:
```suggestion
help: consider changing this argument to be a mutable reference in the `impl` method and the `trait` definition
```

nit: maybe some wording changes (anyway, "referencein" missing a space)

Reply:
I'm having trouble distinguishing if the local comes from a binding or an argument. I checked relying on looking at `local_decl.source_info.scope` being the `0` scope, but that doesn't work at least for the case of literals like `let foo = &16;` (but seemed to work for all other cases).

Reply:
It's probably fine to leave the wording as is then